Doc/reference/expressions.rst

```diff
@@ -740,7 +740,8 @@ which are used to control the execution of a generator function.
    because there is no yield expression that could receive the value.
 
 
-.. coroutinemethod:: agen.athrow(type[, value[, traceback]])
+.. coroutinemethod:: agen.athrow(value)
+                     agen.athrow(type[, value[, traceback]])
```

Comment:
In 3.12 it's documented like this.

Also it can still be called with just the type or just type and value so I think it's correct to keep the `[]`